www/src/css/lib/vanilla-overrides.scss

```diff
@@ -5,4 +5,50 @@
 // If any styles from vanilla framework need to be overridden, corresponding
 // SCSS rules should be put in this file (not mixed with other styles).
 
+// scss/modules/_header.scss
+.banner .nav-primary ul li:hover {
+  background-color:#f7f7f7;
+}
 
+.banner .nav-primary ul li a,
+.banner .nav-primary ul li a:link,
+.banner .nav-primary ul li a:visited {
+  background-color:transparent;
+  font-size:initial;
+  border-left:0;
+  padding:14px 12px 8px 12px;
+}
+
+.banner .nav-primary ul li a:hover,
+.banner .nav-primary ul li a:link:hover,
+.banner .nav-primary ul li a:visited:hover {
+  background-color:transparent;
+}
+
+@media screen and (max-width: 640px) {
```

Comment:
@stevenwilkin You are right. I guess it would make sense to import vanilla-includes here to make the most of variables, etc.

Or maybe it would make more sense to import `vanilla-overrides` into `vanilla-includes` - so that vanilla-includes would be one file to wrap all things vanilla.